src/classes/mech/ActiveState.ts

```diff
@@ -168,13 +172,13 @@ class ActiveState {
   }
 
   public get Move(): number {
-    if (this._pilot_mounted && this._mech.IsShutDown) return 0
-    return !this._pilot_mounted ? this._pilot_move : this._mech.CurrentMove
+    if (this._pilot_mounted && this.ActiveMech.IsShutDown) return 0
```

Comment:
Please see the comment on the ActiveMech function for more information about these replacements.
https://github.com/massif-press/compcon/pull/1612/files#diff-5cb08f43fba64fa933f78e6c6f184d521ded9bb7f065946391e9ab4ff4ac9b12R413